crdb/common.go

```diff
@@ -58,31 +66,35 @@ func ExecuteInTx(ctx context.Context, tx Tx, fn func() error) (err error) {
 	const maxRetries = 50
 	retryCount := 0
 	for {
-		released := false
-		err = fn()
-		if err == nil {
+		releaseFailed := false
+		if err = fn(); err == nil {
 			// RELEASE acts like COMMIT in CockroachDB. We use it since it gives us an
 			// opportunity to react to retryable errors, whereas tx.Commit() doesn't.
-			released = true
 			if err = tx.Exec(ctx, "RELEASE SAVEPOINT cockroach_restart"); err == nil {
 				return nil
 			}
+			releaseFailed = true
 		}
-		// We got an error; let's see if it's a retryable one and, if so, restart.
+
+		// We got an error (i.e. err != nil) if at this point
```

Comment:
nit: this comment needs to be fixed

Reply:
Not 100% sure what you mean, but put it back to exactly what was there. 👍